python_modules/dagster/dagster/_core/definitions/auto_materialize_policy.py

```diff
@@ -69,30 +69,43 @@ class AutoMaterializePolicy(
 ):
     """An AutoMaterializePolicy specifies how Dagster should attempt to keep an asset up-to-date.
 
-    There are two main kinds of auto-materialize policies: eager and lazy. In essence, an asset with
-    an eager policy will try to immediately materialize after upstream changes, while an asset with
-    a lazy policy will only materialize when necessary in order to satisfy the relevant
-    FreshnessPolicies.
+    Each policy consists of a set of `AutoMaterializeRule`s, which are used to determine if there is
+    some reason that an asset / partition of an asset should be auto-materialized, and if so, if
+    there is some reason that an asset / partition of an asset should not be auto-materialized.
```

Comment:
I think we can restructure this to be easier to read - perhaps something like:

> Each policy consists of a set of `AutoMaterializeRule`s, which are used to determine whether an asset or a partition of an asset should or should not be auto-materialized.

Thoughts?